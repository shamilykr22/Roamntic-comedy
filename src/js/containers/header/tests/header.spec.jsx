import React from 'react';
import renderer from 'react-test-renderer';

describe('containers/header/header', () => {
  it('has correct default options', () => {
    const Header = require('../header').default; // eslint-disable-line global-require
    const component = renderer.create(
      <Header />
    );
    const divWrapperComponent = component.toJSON();
    expect(divWrapperComponent).toMatchSnapshot();
  });
});
