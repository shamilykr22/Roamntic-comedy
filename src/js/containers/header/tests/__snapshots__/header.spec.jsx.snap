// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/header/header has correct default options 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#171717",
      "display": "flex",
      "flexDirection": "row",
      "height": 500,
    }
  }
>
  <div
    style={
      Object {
        "width": "10%",
      }
    }
  >
    <img
      className=""
      data-css-1168dpr=""
      src="assets/icons/Back.png"
    />
  </div>
  <div
    style={
      Object {
        "color": "white",
        "fontSize": 40,
        "marginTop": 30,
        "textAlign": "start",
        "width": "85%",
      }
    }
  >
    Romatic comedy
  </div>
  <div
    style={
      Object {
        "width": "5%",
      }
    }
  >
    <img
      className=""
      data-css-t2hlpo=""
      src="assets/icons/search.png"
    />
  </div>
</div>
`;
